api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.7.x.xml

```diff
@@ -19,11 +19,19 @@
 		http://www.liquibase.org/xml/ns/dbchangelog 
 		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 
-	<!--
-		The empty change set is needed for integration tests. It can be removed if this file contains other change sets.
-	-->
-	<changeSet id="42ddb873-f064-4418-a6c0-6c09426c4832" author="wschlegel">
-		<comment>Empty change set for integration tests.</comment>
+	<changeSet id="TRUNK-6204-2023-10-23" author="manojll">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="orders"/>
```

Comment:
This table name does not look correct. 😊

Reply:
@dkayiwa I corrected the table name